bin/i18n/redact_restore_utils.rb

```diff
@@ -111,7 +111,7 @@ def self.redact(source, dest, plugins=[], format='md')
 
     source_data =
       if File.extname(source) == '.json'
-        JSON.parse(File.read(source))
+        JSON.load_file(source)
```

Comment:
Are there any behavior differences between these two ways of loading the JSON data?

Reply:
It practically does the same:
<img width="586" alt="Screenshot 2023-07-14 at 19 07 39" src="https://github.com/code-dot-org/code-dot-org/assets/11708250/1c211542-eee6-4902-8a22-1434738349d7">